exopy_hqc_legacy/instruments/drivers/visa/tektro_awg.py

```diff
@@ -506,6 +590,21 @@ def to_send(self, name, waveform):
         self._driver.write_binary_values(header, waveform, datatype='B')
         self.write('*WAI')
 
+
+    @secure_communication()
+    def send_load_awg_file(self, awg_file, filename = 'setup'):
+        """Command to send and load and .awg file into the AWG
+                awg_file = bytearray
+
+        """
+        name_str = 'MMEMory:DATA "{}",'.format(filename+'.awg').encode('ASCII')
```

Comment:
This is basically a ieee binary block. Pyvisa provides `write_ascii_values` for this use case no need to reinvent the wheel.